_ext/eclipse-base/README.md

```diff
@@ -0,0 +1,66 @@
+# spotless-eclipse-base
+
+Eclipse formatters are part of the the Eclipse User Interface implementations. Hence their public interfaces are depending on various Eclipse modules.
+
+Spotless provides its own plugin framework with `com.diffplug.spotless.JarState`. This allows Spotless the usage of different Eclipse versions in parallel.
+
+
+The `com.diffplug.gradle.spotless:spotless-eclipse-base` artifact mocks the redundant Eclipse OSGI/plugin framework for Spotless. Furthermore it provides Eclipse services adapted for Spotless, which avoids for example the creation of a permanent workspace and reduces the dependencies on Eclipse modules unused by the Eclipse code formatters.
```

Comment:
s/Spotless/Spotless's own use/ ?

Reply:
s/workspace/Eclipse workspace/ ?

I think this makes it more explicit that "workspace" refers to an Eclipse workspace in this context rather than something else.

Reply:
s/the dependencies/dependencies/ ?